crates/kitsune_p2p/fetch/src/pool.rs

```diff
@@ -274,49 +395,12 @@ impl State {
 
     /// The heading to go along with the summary
     #[cfg(any(test, feature = "test_utils"))]
+    #[allow(dead_code)]
     pub fn summary_heading() -> String {
         format!("{:10}  {:>6} {:>6} {}", "key", "#src", "last", "size")
     }
 }
 
-/// A mutable iterator over the FetchPool State
-pub struct StateIter<'a> {
```

Comment:
This was always being `collect`ed and would have been complicated to integrate with these changes so it's just a loop with a filter now